src/commands/createStaticWebApp/postCreateStaticWebApp.ts

```diff
@@ -32,25 +32,31 @@ export async function postCreateStaticWebApp(swaNode: StaticWebAppTreeItem): Pro
 
             while (!deployActionNode) {
                 await productionEnv.actionsTreeItem.refresh(context);
-                const actionTreeItems: ActionTreeItem[] = <ActionTreeItem[]>(await productionEnv.actionsTreeItem.loadAllChildren(context));
-                const filteredTreeItems: ActionTreeItem[] = actionTreeItems.filter(ti => { return ti.data.status !== Status.Completed; }); // only looking at on-going or queued jobs
+                try {
```

Comment:
Any particular reason line 34 isn't in the `try`? Seems like you'd want everything from the loop (except the timeout error) in there

Reply:
I didn't think that it'd be the source of an error, but you're right, may as well shove it in there as well